core/base-service/redirector.js

```diff
@@ -0,0 +1,66 @@
+'use strict'
+
+const emojic = require('emojic')
+const BaseService = require('./base')
+const {
+  serverHasBeenUpSinceResourceCached,
+  setCacheHeadersForStaticResource,
+} = require('./cache-headers')
+const { prepareRoute, namedParamsForMatch } = require('./route')
+const trace = require('./trace')
+
+module.exports = function redirector({ category, route, target }) {
+  return class Redirector extends BaseService {
+    static get category() {
+      return category
+    }
+
+    static get route() {
+      return route
+    }
+
+    static get isDeprecated() {
+      return true
```

Comment:
Will we need to register all the redirected services in the deprecated services list?

Reply:
That list doesn't do anything. Those dates are useful to have readable in code, because it could let us write tests that remind us to downgrade or remove them. Though I think the dates could be placed in the call to `deprecatedService` as easily.

Reply:
Would it be worth writing down redirected services somewhere in a similar fashion to keep track of them?

Reply:
I suppose we could stop supporting the old routes after a good while. In that case having the date we started redirecting would be helpful.

I'd suggest we do that inline by throwing another parameter in the function call.

Reply:
I'm good to approve this, can wait if we wanna do that though

Reply:
Have just added dates.